### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by markburns**
This should hopefully reduce some of the noise from the hound CI bot.

I had a quick guess at which rules the bot was suggesting and pulled those into the config.

Of course you can tweak these, and re-run `bundle exec rubocop --auto-correct`

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  DisplayStyleGuide: true
+  Exclude:
+    - spec/**/*.rb
+    - test_rails_4_app/**/*.rb
+Rails:
+  Enabled: true
+
+
+Metrics/LineLength:
+  Max: 80
```
Comment (zeisler) [inline .rubocop.yml]: I like this at 120

File: lib/active_mocker/mock_creator.rb [hunk 2]
```diff
@@ -45,14 +46,14 @@ def completed?
...
     attr_reader :file,
-                :file_out,
-                :schema_scrapper,
-                :template_creator,
-                :class_introspector,
-                :enabled_partials,
-                :klasses_to_be_mocked,
-                :active_record_base_klass,
-                :mock_append_name
+      :file_out,
```
Comment (zeisler) [inline lib/active_mocker/mock_creator.rb]: don't like this one.

File: lib/active_mocker/mock_creator.rb [hunk 3]
```diff
@@ -227,21 +224,20 @@ def scope_methods
...
 
     module DefinedMethods
-
       def instance_methods
-        class_introspector
-          .get_class
-          .public_instance_methods(false)
-          .sort
-          .map { |m| create_method(m, :instance_method) }
+        class_introspector.
```
Comment (zeisler) [inline lib/active_mocker/mock_creator.rb]: I like the period at the start of the new line.

**Comment by zeisler** [comment]
Once my comments as addressed I would gladly merge. Thanks?

File: lib/active_mocker/mock_creator.rb (hunk 3 above)
Comment (markburns) [inline lib/active_mocker/mock_creator.rb]: You may have already thought about this, but I'll just play the devil's advocate to see what you think:

```ruby
class Maybe
  def method_missing(*args)
    self.class.new
  end
end

m = Maybe.new

m.some_line. #you can put comments in
  #this comment is also possible
  some_continuation.
  #and this one too
  some_more()

m.some_line #you can do this comment but
  #this comment is a syntax error
  .some_continuation
  #and this one too
  .some_more()

```

I'll switch them back to start of line if you still prefer that, as I'd rather spend time on code and design concepts than syntax, but just wanted to surface this point about start of line periods.

File: lib/active_mocker/mock_creator.rb (hunk 2 above)
Comment (markburns) [inline lib/active_mocker/mock_creator.rb]: Neither do I. I'll work out which rule does that

**Comment by markburns** [comment]
OK rerun with those suggestions 😄

**Comment by zeisler** [comment]
@markburns It needs manually rebase off of master.

**Pull request merged**